images/hubble-proto/Makefile

```diff
@@ -0,0 +1,12 @@
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+include Makefile.defs
+
+.PHONY: image
+image:
+	docker build -t cilium/hubble-proto:${version} .
+
+.PHONY: push
+push: image
+	docker push cilium/hubble-proto:${version}
```

Comment:
I'm OK with having a new `Makefile` here but for consistency `images/Makefile` should also have a target to build `hubble-proto`.

Reply:
that Makefile is rather complex and does docker in docker builds for images we actually ship to users. This image is only used by developers

Reply:
@glibsm  `images/Makefile` is not only for users. It also contains developer-only images such as `cilium-builder`.

https://github.com/cilium/cilium/blob/10f584d33cecff4e542d57d53a920427cb124296/Makefile.docker#L120-L126

I really don't see a difference between `cilium-builder` and `hubble-proto` in the way they are used. Plus the existing Makefile also makes sure we have ARM64 support, which our `hubble-proto` Makefile currently misses.

Reply:
> Plus the existing Makefile also makes sure we have ARM64 support, which our hubble-proto Makefile currently misses.

The `hubble-proto` image is only used to generate files from proto. Why would the CPU architecture matter for this purpose?

Reply:
So people can develop (including updating the protobuf via `make -C api/v1`) on ARM64 machines? Running `make generate-api` to generate the Swagger API Go code already works on ARM64 as of today, which conceptually very very similar to `make -C api/v1`.

Please do not misunderstand me: It's a very niche use-case and I don't think ARM64 support should be a blocker on this PR at all. The existing Makefile might be complex, but it is part of a well-maintained process. Adding us there also make sure that non-Hubble project contributors (such as e.g. the ARM64 folks) will make sure our Docker images get updated whenever there are changes to our docker build infra (such as adding support for new architectures, moving docker registries, adding e.g. vulnerability scanner tooling and so forth)